parlai/core/torch_generator_agent.py

```diff
@@ -564,7 +585,7 @@ def _init_cuda_buffer(self, batchsize, maxlen, force=False):
         if self.use_cuda and (force or not hasattr(self, 'buffer_initialized')):
             try:
                 self._control_local_metrics(disabled=True)
-                loss = self.compute_loss(self._dummy_batch(batchsize, maxlen))
+                loss = 0 * self.compute_loss(self._dummy_batch(batchsize, maxlen))
```

Comment:
was this causing a problem before, or is this just for extra clarity?

Reply:
Extra clarity.